src/Microsoft.ML.OnnxTransform/OnnxTransform.cs

```diff
@@ -187,18 +218,68 @@ private static IRowMapper Create(IHostEnvironment env, ModelLoadContext ctx, Sch
             _args = args;
         }
 
-        public OnnxTransform(IHostEnvironment env, string modelFile)
-            : this(env, new Arguments() { ModelFile = modelFile, InputColumns = new string[] { }, OutputColumns = new string[] { } })
+        /// <summary>
+        /// Transform for scoring ONNX models. Input data column names/types must exactly match
+        /// all model input names. All possible output columns are generated, with names/types
+        /// specified by model.
+        /// </summary>
+        /// <param name="env"></param>
+        /// <param name="modelFile">Model file path</param>
+        /// <param name="gpuDeviceId">Optional device ID to run execution on</param>
+        /// <param name="fallbackToCpu">If GPU error, raise exception or fallback to CPU</param>
```

Comment:
I think we tend to place the '.' at the end of the sentences, since they will appear in the user facing documentation website.